CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/panes/DataAccessViewCategoryPanelController.java

```diff
@@ -96,8 +101,22 @@ public boolean isChanged() {
         final Preferences prefs = NbPreferences.forModule(DataAccessViewPreferenceKeys.class);
         final DataAccessViewCategoryPanel panel = getPanel();
         final List<String> hiddenCategory = panel.getHiddenCategory();
-        return (!hiddenCategory.isEmpty())
-                || (!prefs.get(DataAccessViewPreferenceKeys.HIDDEN_DA_VIEW, DataAccessViewPreferenceKeys.HIDDEN_DA_VIEW_DEFAULT).isEmpty());
+        final List<String> visibleCategory = panel.getVisibleCategory();
+
+
+        if (reorderButtonPressed) {
+            if (!visibleCategory.equals(visibleNow)) {
+                orderChanged = true;
+                visibleNow = panel.getVisibleCategory();
+            } else {
+                orderChanged = false;
+                reorderButtonPressed = false;
+            }
```

Comment:
Remove the **else** block. Reset these variables after changes have been applied.